bin/app-server.sh

```diff
@@ -160,7 +160,7 @@ fi
 #Determined log file.  Run node appropriately.
 cd ../lib
 
-export ZOWE_LIB_DIR=$(cd `dirname $0` && pwd)
```

Comment:
This change works for me, and I get same directories with either one. I'm not sure what edgecase this was trying to account for

Reply:
Edge case is when script is executed from different folder than $0 will be different
This changes removes $0 as we are already in lib